src/iam.ts

```diff
@@ -473,9 +473,15 @@ export class IAM extends IAMBase {
    * @returns return issued token
    *
    */
-  async issuePublicClaim({ token }: { token: string }) {
+  async issuePublicClaim({ token, publicClaim }: { token?: string, publicClaim?: IPublicClaim }) {
```

Comment:
It looks like this method doesn't needed as all it does just forward call to this._issueClaims.issuePublicClaim

Reply:
We will also need to clear `token` from user data on cache server when it receive issuance claim

Reply:
Although it can be kept if later will need to change way of issuing public claim